internal/pkg/manifest/backend_svc.go

```diff
@@ -28,17 +28,15 @@ type BackendServiceProps struct {
 
 // BackendService holds the configuration to create a backend service manifest.
 type BackendService struct {
-	Service      `yaml:",inline"`
-	Image        imageWithPortAndHealthcheck `yaml:",flow"`
-	TaskConfig   `yaml:",inline"`
-	LogConfig    `yaml:"logging,flow"`
-	Sidecar      `yaml:",inline"`
-	Environments map[string]*backendServiceOverrideConfig `yaml:",flow"`
+	Service              `yaml:",inline"`
+	BackendServiceConfig `yaml:",inline"`
+	Environments         map[string]*BackendServiceConfig `yaml:",flow"`
```

Comment:
Should we add a comment here explain why this one is a pointer, whereas the embedded struct isn't?

Reply:
done!